translate/storage/mozilla_lang.py

```diff
@@ -121,22 +136,24 @@ def parse(self, lines):
 
             if line.startswith(';'):
                 source_unit = self.addsourceunit(line[1:])
+                source_unit.eol = self.eol
                 source_unit.addlocation(
                     "%s:%d" % (self.filename, lineoffset + 1))
                 if comment is not None:
                     source_unit.addnote(comment[:-1], 'developer')
                     comment = ""
 
     def serialize(self, out):
+        eol = self.eol.encode('utf-8')
         if self.is_active or self.mark_active:
-            out.write(b"## active ##\n")
+            out.write(b"## active ##")
+            out.write(byte(self.eol))
```

Comment:
If you refer here to `self.eol` then https://github.com/translate/translate/pull/3498/files#diff-0b68326f21f831b4685b4ee07436396aR147 is unused. As stated in https://github.com/translate/translate/pull/3498/files#r80911497 I would move these changes to `self.eol` to `__init__` so they are run once. It will also reduce the diff.